translate/src/modules/history/components/HistoryTranslation.tsx

```diff
@@ -196,17 +196,23 @@ export function HistoryTranslationBase({
 
   const review = {
     id: 'history-translation--unreviewed',
-    vars: { user: '', reviewedDate: new Date(translation.dateIso) },
+    vars: { user: '', reviewedDate: new Date() },
```

Comment:
Can we completely drop `reviewedDate` here?

Reply:
Potentially... is there a way for me to pass `approvedDate` or `rejectedDate` to the .ftl file if we do remove this variable?

Reply:
Ah, I missed that this was part of an FTL message.

Reply:
Can we use `''` instead of `new Date()`?

Reply:
I seem to get a type error if I try to switch to `''`. Should I still try to move forward with using `''` instead of `new Date()`?

<img width="448" alt="Screenshot 2024-07-03 at 1 53 47 PM" src="https://github.com/mozilla/pontoon/assets/90732381/7fa3979b-6a1a-493e-bb74-b22f1e02026c">

Reply:
Makes sense. Let's keep `new Date()`.